combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
+    @cycle_rituals = @ritual_type == "cycle"
```

Comment:
You don't need to initialize it before assigning to it. The `false` will be immediately overwritten. If the ritual type is _not_ 'cycle', then it will still do the assignment, but with a `false` value (i.e. `@cycle_rituals` will always have a value after the comparison is done).